examples/apps/popup/src/main.rs

```diff
@@ -52,11 +52,15 @@ fn render(frame: &mut Frame, show_popup: bool) {
     frame.render_widget(Block::bordered().title("Content").on_blue(), content);
 
     if show_popup {
-        let popup = Block::bordered().title("Popup");
+        let popup_block = Block::bordered().title("Popup");
         let popup_area = centered_area(area, 60, 20);
```

Comment:
how about using the 0.30 centering method on rect here https://docs.rs/ratatui/latest/ratatui/prelude/struct.Rect.html#method.centered

Reply:
fixed

![popup](https://github.com/user-attachments/assets/e2516d62-5c4d-4f4a-b08f-e32521fcaa0c)